jest.setup.ts

```diff
@@ -0,0 +1 @@
+import "@inrupt/jest-jsdom-polyfills";
```

Comment:
```suggestion
import "@inrupt/jest-jsdom-polyfills";

```

missing trailing new line, and eslint may complain about license header.